service/experiment-requests.yaml

```diff
@@ -16,24 +16,10 @@
 # You can run "make presubmit" to do basic validation on this file.
 # Please add new experiment requests towards the top of this file.
 
-- experiment: 2020-09-26
+- experiment: 2020-09-20
```

Comment:
Can you please rebase after https://github.com/google/fuzzbench/pull/752, this needs to be 2020-09-24